pandas/tseries/common.py

```diff
@@ -106,6 +106,8 @@ def _delegate_property_get(self, name):
         elif not is_list_like(result):
             return result
 
+        result = np.asarray(result)
+
```

Comment:
hmm, why are you converting back to an ndarray here? I don't think this necessary